examples/commands/message-stores.php

```diff
@@ -103,6 +103,9 @@
     new DropStoreCommand(new ServiceLocator($factories)),
 ]);
 
+$clock = new MonotonicClock();
+$clock->sleep(10);
```

Comment:
Is this the solution now?
> Looks like OS is too slow during its boot phase, I'm looking for a solution 🙂

Reply:
At least, the only one that I found that isn't triggering errors due to OS slow start.

Calling `http://127.0.0.1:9200` is not enough as it's only a health response, OS is not fully ready when the health response is OK.